lib/ansible/plugins/lookup/password.py

```diff
@@ -136,7 +136,7 @@ def _gen_candidate_chars(characters):
     return chars
 
 
-def _random_password(length=DEFAULT_LENGTH, chars=C.DEFAULT_PASSWORD_CHARS):
+def random_password(length=DEFAULT_LENGTH, chars=C.DEFAULT_PASSWORD_CHARS):
```

Comment:
You were wondering where to move this.  Let's move it to lib/ansible/utils/encrypt.py or a new file in lib/ansible/utils (password.py or similar).